src/test/groovy/com/example/EndpointSpec.groovy

```diff
@@ -0,0 +1,60 @@
+package com.example
+
+import org.springframework.http.HttpStatus
+import spock.lang.Unroll
+
+class EndpointSpec extends IntegrationSpec {
+
+    def "should return list of cards with three elements"() {
+        when:
+        def response = restTemplate.getForEntity(localUrl("/get-cards"), Card[])
+
+        then:
+        response.statusCode == HttpStatus.OK
+        response.body.size() == 3
```

Comment:
No jak będzie mongo to wtedy możemy w given dodać jakieś rzeczy do bazy - wtedy w then nie będzie zaskakująca liczba 3 :)